pkg/operator/certrotationcontroller/satokensigner_controller.go

```diff
@@ -189,6 +190,11 @@ func (c *SATokenSignerController) syncWorker() error {
 		return err
 	}
 	needNewSATokenSigningKey := errors.IsNotFound(err) || len(saTokenSigner.Data["service-account.key"]) == 0 || len(saTokenSigner.Data["service-account.pub"]) == 0
+	matchingKeyPair, err := isValidKeyPair(saTokenSigner.Data["service-account.pub"], saTokenSigner.Data["service-account.key"])
```

Comment:
Is it desirable to perform the check for length separate from the validity check?

Reply:
Yes. you are right. The check is not needed now. I removed it!